odbc-api/tests/integration.rs

```diff
@@ -2519,3 +2519,15 @@ fn many_diagnostic_messages() {
 
     // We do not have an explicit assertion, we are just happy if no integer addition overflows.
 }
+
+
+#[test_case(MSSQL; "Microsoft SQL Server")]
```

Comment:
Nice. Out of curiosity: Do you have any trouble with the docker test setup and the development container. I assume `Sqlite` and `MariaDB` are missing for a reason. If so don't worry I'll add them myself

Reply:
Nope, I can add these. I didn't notice any examples of parameterized tests with expected values so that's why I skipped it initially, but it looks like `test_case` does support that